### ISSUE TITLE
Merging the community

### ISSUE THREAD

**Issue by l3pp4rd** [vilniusgolang/website#5]
Since we currently have two branches of **golang** user groups I would suggest merging into a single group aka golang Lithuania where all active golang community users could have access to post events and be close to other gophers.

What do you think @ernestas-poskus if we move everything under golang lithuania user group? we could create a separate organization like golang-lt or keep gopherslt. merge websites and tag posts with city if needed.

**Comment by ernestas-poskus** [comment]
I agree this should be resolved. Something like Golang-Lithuania or golang-lt sounds nice.

**Comment by l3pp4rd** [comment]
and concerning website we can merge posts and use web githook to autoupdate on push on our servers, the references to vilniusgolang.lt and gophers.lt should remain, but host the same website. golang.lt was already bought few years ago for commercial reasons it seems.

**Comment by l3pp4rd** [comment]
have created https://github.com/golang-lt @justinas and @ernestas-poskus are invited. All members without exception gain admin access rights by default. Invite other gophers on your preference

**Comment by ernestas-poskus** [comment]
Excellent. I will transfer some repositories to new organization.

**Comment by justinas** [comment]
Some news on the venue (finally!): we'll be gathering at VU MIF, Naugarduko st. 24. The date is February 17th and I propose starting at 18:30.

I'm preparing the post for vilniusgolang.lt. @ernestas-poskus could I get the permissions to post on meetup.com?

**Comment by l3pp4rd** [comment]
I will be away until next week, when I'm back we could both merge the website

**Comment by ernestas-poskus** [comment]
@l3pp4rd I have transferred 'presentations' repo, so now we must decide what web domain we will keep

**Comment by l3pp4rd** [comment]
we can keep both, and deploy to both. I have made a github webhook to autodeploy **gophers.lt** on push to master. You can anytime update the contents of website with new features same as everyone as a member has full access to all repositories

**Comment by ernestas-poskus** [comment]
I think keeping both vilniusgolang.lt and global gophers.lt will confuse communities and will create feeling of separateness.

**Comment by ernestas-poskus** [comment]
I would like to deprecate/remove vilniusgolang repo and do everything in golang-lt

**Comment by l3pp4rd** [comment]
in that case gophers.lt seems more like a general purpose domain. you can make 301 redirect

**Comment by l3pp4rd** [comment]
awesome, I think we have sorted everything out, glad it is now a general golang lithuania user group..

**Comment by l3pp4rd** [comment]
I see you had more functionality in website, like google analytics(not sure whether we need this) but feel free as everyone else to edit the content or migrate some features you had in website..

**Comment by ernestas-poskus** [comment]
I think google analytics is nice addition to track how we are doing. I will redirects and further migration this week.

**Comment by l3pp4rd** [comment]
I've made the github webhook to autodeploy to gophers.lt and github.io pages. So I think it will be very convenient for us to work with.. you can add GA analytics if you want, might become interesting someday :) everytime you push to master it will be deployed.

**Comment by ernestas-poskus** [comment]
Added post about merging of communities http://vilniusgolang.lt/posts/merging-of-communities/

Will do the PR for some extra functionality for http://gophers.lt/ like iCal calendar for meetup and links to twitter and etc.

**Comment by l3pp4rd** [comment]
great @ernestas-poskus all members can edit the site in way they see fit. its great that we finally merged it..
